packages/types/src/storage-types.ts

```diff
@@ -125,6 +125,7 @@ export enum EthereumNetwork {
   MAINNET = 1,
   RINKEBY = 4,
   KOVAN = 42,
+  XDAI = 100,
```

Comment:
Same here, We should add `sokol` here as xdai testnet.

Reply:
I don't think it's mandatory actually, what do you expect to test specifically with `sokol` that would behave differently with `rinkeby`?

Reply:
got the idea , in the end i stayed with xdai only